pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/test_inputs/wgs/Scientific/G94794.CHMI_CHMI3_WGS2.json

```diff
@@ -1,7 +1,8 @@
 {
   "ReblockGVCF.gvcf": "gs://broad-gotc-test-storage/germline_single_sample/wgs/scientific/truth/{TRUTH_BRANCH}/G94794.CHMI_CHMI3_WGS2/CHMI_CHMI3_WGS2.f6c39eb5-0716-4f02-9be8-47b343c5830e.g.vcf.gz",
   "ReblockGVCF.gvcf_index": "gs://broad-gotc-test-storage/germline_single_sample/wgs/scientific/truth/{TRUTH_BRANCH}/G94794.CHMI_CHMI3_WGS2/CHMI_CHMI3_WGS2.f6c39eb5-0716-4f02-9be8-47b343c5830e.g.vcf.gz.tbi",
+  "ReblockGVCF.calling_interval_list": "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.interval_list",
```

Comment:
You removed {TRUTH_BRANCH} from line 2 in other files like, G96830.NA12878.json.  Did you mean to do that here too, or no?

Reply:
Yes, I'm making the naming convention more consistent so it's easier to programmatically check the scientific test results.

Reply:
Yeah, I will have to update the scientific data in the same way as I did for plumbing.  I probably ought to do that now even though the scientific tests don't have to pass to merge to develop.

Reply:
The {TRUTH_BRANCH} is gone because I'm intentionally decoupling the reblocking from the single-sample pipeline outputs.  No test outputs a non-reblocked GVCF, so rather than using a truth file as input, this needs an independent input.